multi-agent/run/scripts/migrate_builtin_system.py

```
@@ -0,0 +1,445 @@
+    4. ``$ENCRYPTION_KEY`` env var (legacy / backward-compat name).
+
+    Returns ``""`` if nothing is found.
+    """
+    try:
+        from config.app_config import AppConfig
+        key = AppConfig().credential_encryption_key
+        if key:
+            logger.info("Auto-discovered encryption key from AppConfig.")
+            return key
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)